src/app/clusters/level-control/level-control.cpp

```diff
@@ -568,36 +572,37 @@ static EmberAfStatus moveToLevelHandler(EndpointId endpoint, CommandId commandId
     // logic is to prevent a light from transitioning from off to bright to dim.
     // Instead, a light that is off will stay off until the target level is
     // reached.
-    if (currentLevel <= state->moveToLevel)
+    if (currentLevel.Value() <= state->moveToLevel)
```

Comment:
Will crash if stored level was null?  Same for a bunch of the other callsites here...

Reply:
@bzbarsky-apple 
Theoretically, it is possible that a currentLevel might be null but in reality that case should never happen since the level is set the first time device boots up if i am not mistaken.

I'll add a check at the start of the handler function to make sure the `currentLevel` is not null, but the spec doesn't specify a behavior in this scenario.

Reply:
Same issue for the `emberAfLevelControlClusterServerInitCallback`, what happens if `startUpCurrentLevel` is null? Do we assume that since `currentLevel` is null we should set it's value to it's min allowed value?